PayPalNativePayments/src/main/java/com/paypal/android/paypalnativepayments/PayPalNativeShippingListener.kt

```diff
@@ -0,0 +1,29 @@
+package com.paypal.android.paypalnativepayments
+
+/**
+ * An optional listener to receive notifcations if the user changes their shipping information.
+ */
+interface PayPalNativeShippingListener {
+
+    /**
+     * Notify when the users selected shipping address changes
+     *
+     * @param actions actions to perform after patching the order
+     * @param shippingAddress the user's most recently selected shipping address
+     */
+    fun onPayPalNativeShippingAddressChange(
+        actions: PayPalNativeShippingActions,
+        shippingAddress: PayPalNativeShippingAddress
+    )
+
+    /**
+     * Notify when the users selected shipping method changes
+     *
+     * @param actions actions to perform after patching the order
```

Comment:
Can we make this docstring more detailed? A few things it should answer:

* Is using `actions` required?
* When should I use these `actions`?

Reply:
For example, aren't they going to need to do a patch `add` with all of the shipping methods every time this callback is invoked in order for the UI to update appropriately? (This was the original issue that we brought up to the NXO team, and I started to address [here](https://github.com/paypal/iOS-SDK/pull/136))

Reply:
sure, ill improve the doc string to add more details description